src/main/resources/db/migration/V017__UpdateRecordingStatusType.sql

```diff
@@ -0,0 +1,3 @@
+DROP TYPE public.ACCESS_STATUS CASCADE;
```

Comment:
Since PR has opened master has had another migration merged in. This file need to be renamed to V018__...